CIP-XXXX/README.md

```diff
@@ -0,0 +1,230 @@
+---
+CIP: ?
+Title: Extensive Light Wallet Standard
+Category: Wallets
+Status: Proposed
+Authors:
+    - Vladimir Kalnitsky <vladimir@mlabs.city>
+Implementors: []
+Discussions:
+    - https://github.com/cardano-foundation/cips/pulls/620
+    - [Cardano Working Groups Discord server](https://discord.gg/XFjAzsQyaM)
```

Comment:
The YAML section here doesn't understand Markdown.  The invite link should be OK on its own because people following it will have no trouble finding the `#wallet-standard` channel.